platform/ios/.swiftlint.yml

```diff
@@ -0,0 +1,24 @@
+disabled_rules:
```

Comment:
I think it is possible to use `only_rules` to specify which rules should be enabled, instead of saying which rules should be disabled.

Reply:
I preferred to take this approach, so that we can later eliminate them one by one (or two by two) in separate PRs... TBH. But if you think that's a better approach, I pretty much know which rules I fixed in this PR, I can put them there if needed

Reply:
Well I don't know, yet, if we should use all of the SwiftLint rules. And `only_rules` is explicit as to what rules we are using, instead of what we are not using.

How is it for the other languages, like JS, Kotlin, TS, are we using all of the default lint rules so we follow community standards? Or we imply some of our own reasoning/liking as to what rules should be added? It will be nice to be consistent with other languages.

Reply:
Generally, yes. We pretty much worked with Kotlin in the same way. ES was already here when I arrived, though, but looks like it was done in the same way (at least looking at how we've disabled them recently.

Reply:
Ok then, we are good to go with all the rules. And disable nasty ones when needed.